feature/announcement/src/test/java/io/github/droidkaigi/confsched2019/announcement/ui/store/AnnouncementStoreTest.kt

```diff
@@ -22,7 +22,7 @@ class AnnouncementStoreTest {
 
     @Before fun setUp() {
         MockKAnnotations.init(this, relaxUnitFun = true)
-        CoroutinePlugin.mainDispatcherHandler = { Dispatchers.Default }
+        CoroutinePlugin.mainDispatcherHandler = { TestActionDispatcher() }
```

Comment:
mainDispatcherHandler will be used to process events of LiveData. So this means processing them will be done on the current thread. i.e. the test thread